src/proposal.mligo

```diff
@@ -193,28 +208,27 @@ let unfreeze_proposer_and_voter_token
   (rejected_proposal_slash, is_accepted, proposal, period, fixed_fee, store :
     (proposal * contract_extra -> nat) * bool * proposal * period * nat * storage): storage =
   // unfreeze_proposer_token
-  let (tokens, store) =
+  let (tokens, burn_amount, store) =
```

Comment:
`store` is no longer needed here, since it doesn't get updated anymore.

Reply:
Updated.